ocs_ci/ocs/resources/noobaa.py

```diff
@@ -81,6 +81,17 @@ def s3_get_all_bucket_objects(self):
         """
         return [bucket for bucket in self._s3_resource.buckets.all()]
 
+    def s3_verify_bucket_exists(self, bucket):
+        """
+        Args:
+            bucket: The bucket object to be verified
+
+        """
+        if bucket in self._s3_resource.buckets.all():
```

Comment:
You're accessing a private member, but I actually changed it to be publicly accessible on my PR, so we'll have to fix that later on.

This might take a long time and depends on the amount of buckets on the server. The documentation recommends this method:
```suggestion
        try:
          self._s3_resource.meta.client.head_bucket(Bucket=bucket.name)
          logger.info(f"{bucket.name} exists"}
        except ClientError:
          logger.info(f"{bucket.name} does not exist"}
```

Reply:
True, will test this. Thanks :)